trackintel/preprocessing/positionfixes.py

```diff
@@ -364,7 +386,24 @@ def generate_triplegs(pfs_input, stps_input, method="between_staypoints", gap_th
         tpls = tpls.set_geometry("geom")
         tpls.crs = pfs.crs
 
-        # check the correctness of the generated tpls
+        # check the correctness of the generated triplegs
```

Comment:
would it be nicer to encapsulate this check in a function? `generate_triplegs()` in general is quite complex and this might help with readability

Reply:
👍